composer.json

```diff
@@ -43,17 +43,20 @@
 		}
 	},
 	"scripts": {
+		"ci": [
+			"composer test",
+			"composer cs",
+			"composer md"
+		],
 		"test": [
 			"composer validate --no-interaction",
 			"phpunit"
 		],
 		"cs": [
```

Comment:
The cs command I added here is intended for all code checking. What tools are used for that is an implementation detail.